### PR TITLE
feat(FE):Terms and Conditions

### PR DESCRIPTION
**Pull request by mehtapy**

### CODE REVIEW

**Comment by albarkauss** [comment]
Looks good! 👍 
Do we need  the `regtest` ? or we can delete that?

**Comment by mehtapy** [comment]
No sorry we dont need the regtest file it was only for testing

File: client/src/components/pages/register/regtest.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              Register
+            </Button>
+          </Form.Item>
+        </Form>
+      </div>
+    </div>
+  );
+};
+
+export default Register;
```
Comment (gitadventurer241) [inline client/src/components/pages/register/regtest.jsx:101]: Can you delete this file ?

**Review by gitadventurer241 — CHANGES_REQUESTED** [review]

File: client/src/components/pages/register/regtest.jsx (hunk 1 above)
Comment (gitadventurer241) [reply client/src/components/pages/register/regtest.jsx:101]: @mehtapy

**Review by albarkauss — APPROVED** [review]

**Pull request merged**